packages/react/src/components/CardGroup/__stories__/data/cards.json

```diff
@@ -44,7 +44,7 @@
   "Images": [
     {
       "image": {
-        "defaultSrc": "https://fpoimg.com/1056x792?text=4:3&bg_color=ee5396&text_color=161616",
+        "defaultSrc": "https://dummyimage.com/1056x792/ee5396/161616%26text=4:3",
```

Comment:
@kennylam weren't we moving away from dummyimage?

Reply:
@jeffchew Yep, I don’t think that PR had been merged when I opened this one. Should be resolved when this branch updates.